com.unity.multiplayer.mlapi/Runtime/Messaging/CustomMessageManager.cs

```diff
@@ -14,8 +14,15 @@ namespace MLAPI.Messaging
     /// The manager class to manage custom messages, note that this is different from the NetworkManager custom messages.
     /// These are named and are much easier to use.
     /// </summary>
-    public static class CustomMessagingManager
+    public class CustomMessagingManager
     {
+        private NetworkManager m_NetworkManager { get; }
```

Comment:
Idea for later refactor, since several classes all have this pattern of having their own m_NetworkManager, you could have a base class like "NetworkManagerBound" or something.  You could then wrap the xtor, m_NetworkManager property, its getter and maybe more things in that.

Reply:
Good input. At least a field would be nice, but some classes already have constructors like noels RPC queues so I think those should be left.